### ISSUE TITLE
Gitako and githubtree are too close to each other

### ISSUE THREAD

**Issue by faimin** [EnixCoda/Gitako#304]
Gitako and githubtree are too close to each other, and it is easy to touch them by mistake.
Suggest adjusting the position of gitako. ❤️

![image](https://github.com/user-attachments/assets/9c1509fb-b08f-482f-85a3-c475992769aa)

**Comment by EnixCoda** [comment]
Thanks, I'll tweak its default position downwards.

Btw, do you know that you can drag Gitako's icon?

**Comment by faimin** [comment]
NO, because the default interaction is whenever the mouse moves to the Gitako logo, the tree will open automatically.

**Comment by EnixCoda** [comment]
> NO, because the default interaction is whenever the mouse moves to the Gitako logo, the tree will open automatically.

My bad. I should have mentioned that this feature is only available when pinned

**Comment by EnixCoda** [comment]
Forgot to close this. Fixed in 0c9acf7a183d52af17fdcc31052bf3ac590ae73d 1 year ago
